src/ProfilesList.json

```
@@ -224,9 +224,10 @@
   "Sairajepawar.json",
   "Haris-Mohanty.json",
   "leabuende.json",
+  "khushminder270.json"
```

```suggestion
  "khushminder270.json",
```

Add comma `,` at last of you file name.